app/Services/BulkService.php

```diff
@@ -259,7 +259,11 @@ public function editing(array $fields, Bulk $bulk): int
         $tagIds = Arr::get($fields, 'tags', []);
 
         if ($this->entityName === 'relations') {
-            return $this->updateRelations($filledFields);
+            $mirrorOptions = [];
+            $mirrorOptions['unmirror'] = request()->unmirror;
```

Comment:
request() still being called in BulkService, so not testable :P `$fields` equals `request()->all()` so we can test on `$mirroOptions['unmirror'] = (bool) Arr::get($fields, 'unmirror', '0');`